README.md

```diff
@@ -59,5 +67,9 @@ For a full example inside an express app, see the [sample](sample) directory wit
 - Made resolveWithFullResponse the default
 
 ## 2.0
-- ES6 syntax, dropping node 5 support.
+- ES6 syntax, dropping Node.js 5 support.
 
+## 3.0
```

Comment:
It looks like you added a breaking change to the signature of the `RequestPromiseMiddlewareFramework` constructor. Can you list that? Personally, in major releases I like to have a big bold section outlining the breaking changes separate from the features.

Reply:
Can it be breaking and non-breaking at the same time? :) I will improve the changelog messaging, but note that the old signature for the constructor will still work.

Reply:
If the old signature still works, then it isn't breaking. (Dropping node6 support kinda is, but I'm not sure I see other libraries versioning for that purpose).